src/wp-admin/includes/ajax-actions.php

```diff
@@ -5409,13 +5417,15 @@ function wp_ajax_health_check_site_status_result() {
  * @see WP_REST_Site_Health_Controller::get_directory_sizes()
  */
 function wp_ajax_health_check_get_sizes() {
+	_deprecated_function( __FUNCTION__, '5.6.0', 'WP_REST_Site_Health_Controller::get_directory_sizes()' );
+
 	_doing_it_wrong(
 		'wp_ajax_health_check_get_sizes',
 		sprintf(
 		// translators: 1: The Site Health action that is no longer used by core. 2: The new function that replaces it.
 			__( 'The Site Health check for %1$s has been replaced with %2$s.' ),
 			'wp_ajax_health_check_get_sizes',
-			'WP_REST_Site_Health_Controller::get_directory_sizes'
+			'WP_REST_Site_Health_Controller::get_directory_sizes()'
 		),
 		'5.6.0'
 	);
```

Comment:
Should the `_doing_it_wrong()` be removed now there will be a `_deprecated_function()` call ? The information provided is essentially the same, so this seems redundant.